src/pc/common/PaletteBiome.js

```diff
@@ -83,7 +82,6 @@ class BiomeSection {
       palette.push(varInt.read(smartBuffer))
     }
 
-    varInt.read(smartBuffer)
```

Comment:
Is this safe to remove, are we expected to exhaust this buffer?

Reply:
I just moved it into the PaletteContainer